syncer/syncer.go

```diff
@@ -1388,11 +1404,11 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 		s.wg.Add(1)
 		name := queueBucketName(i)
 		s.queueBucketMapping = append(s.queueBucketMapping, name)
-		s.workerLagMap[name] = atomic.NewInt64(0)
+		s.workerLagMap[dmlWorkerLagKey(i)] = atomic.NewInt64(0)
 		go func(i int, name string) {
 			ctx2, cancel := context.WithCancel(ctx)
 			ctctx := s.tctx.WithContext(ctx2)
-			s.syncDML(ctctx, name, s.toDBConns[i], s.jobs[i])
+			s.syncDML(ctctx, name, s.toDBConns[i], s.jobs[i], i)
```

Comment:
maybe pass the value of `dmlWorkerLagKey(i)` to avoid call it again inside `syncDML`

Reply:
now **avoid call it again** outside syncDML